bokehjs/src/coffee/renderer/tile/tile_renderer.coffee

```diff
@@ -24,6 +29,33 @@ class TileRendererView extends PlotWidget
     @_last_height = undefined
     @_last_width = undefined
 
+  _add_attribution: () =>
+    attribution = @mget('tile_source').get('attribution')
+
+    if _.isString(attribution) and attribution.length > 0
+      if @attributionEl?
+        @attributionEl.html(attribution)
+      else
+        border_width = @map_plot.get('outline_line_width').value
+        bottom_offset = @map_plot.get('min_border_bottom') + border_width
```

Comment:
So, "resolve conflicts" commit doesn't touch this line, so it's intentional. Still I'm not sure if `min_border_bottom` is the right property to query. Can you use `frame` here, as you did with `right_offset`?